README.md

```diff
@@ -6,6 +6,8 @@
 
 Assim como o Laguinho da UFCG é o ponto mais central da universidade, esta API centralizará os dados utilizados nos repositórios da OpenDevUFCG.
 
+**_OBS: Estamos passando por um processo de reformulação da ideia e do projeto em si, o desenvolvimento do novo laguinho está acontecendo na [branch development](https://github.com/OpenDevUFCG/laguinho-api/tree/development) e as issues relacionadas estão marcadas com a label [`novo laguinho`](https://github.com/OpenDevUFCG/laguinho-api/labels/novo%20laguinho). Para saber mais sobre essa reformulação, veja a [Issue #31](https://github.com/OpenDevUFCG/laguinho-api/issues/31)._**
```

Comment:
Essa issue devia ser substituída com uma wiki... ou design docs

Reply:
Eu acho que não tem problema ser a própria issue, a thread não é grande e ela não vai sumir

Reply:
mas é confusa a issue, e tem alguns vai e voltas

Reply:
assim, pra dizer q tem um novo laguinho é ok. mas pra explicar o laguinho, deveria ser outra coisa

Reply:
É, realmente :thinking: Fica melhor mover pra outro lugar mesmo, ou escrever uma explicação sucinta no próprio readme

Reply:
Concordo, mas acho que pode ser outra issue, dá pra quebrar o galho assim por enquanto, já que a wiki demoraria um pouco mais pra fazer

Reply:
sei... to de olho 👀

Reply:
KKKKKKK vou copiar a que tu criou na CLI aqui, mas deixar esse por enquanto, pq já teve gente que foi abrir o laguinho hj e só notou que tava em reformulação pq abriu uma issue que tava aberta por engano e citava isso, daí esse PR é só emergencial msm KKKKKKKK